### PR TITLE
ISPN-12445 404 small design improvement and unit test

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12445

The PR covers a unit test that mocks and tests the history button click and an override for testing of the languages translation, which is useful to be used on tests and verify labels are correctly externalised into the languages file 

Screenshot 2020-10-22 at 11 54 43

### CODE REVIEW

File: src/app/assets/languages/en.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
     "tutorials":  "Run the simple tutorials",
     "go-to-console":  "Go to the console"
   },
+  "not-found-page" : {
+    "title": "Page not found",
+    "description": "We didn't find a page that matches the address you navigated to.",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:23]:
```suggestion
    "description": "That page no longer exists.",
```

File: src/app/assets/languages/en.json [hunk 2]
```diff
@@ -18,6 +18,11 @@
...
+    "button" : "Take me home"
```
Comment (oraNod) [inline src/app/assets/languages/en.json:24]:
```suggestion
    "button" : "Return to home page"
```

**Review by oraNod — APPROVED** [review]

**Comment by karesti** [comment]
@oraNod your changes integrated

**Comment by oraNod** [comment]
thanks @karesti

**Pull request merged**